README.md

```diff
@@ -1063,26 +1063,42 @@ class MyCharm(ops.CharmBase):
         event.defer()
 
 
-def test_start_on_deferred_update_status(MyCharm):
```

Comment:
Yeah, this line is being removed, because it seems wrong (it's in the current Scenario main docs). `MyCharm` is just a regular class at module level, so it doesn't need to be passed in. It looks like there was going to be a mycharm fixture and then it was changed to a plain class or something?

Anyway, it's gone away in this PR.

Reply:
oh my bad, misread the diff. note to self: don't do hasty reviews late in the day.